server/test/ping.spec.js

```diff
@@ -1,22 +1,110 @@
 const chai = require("chai");
 const chaiHttp = require("chai-http");
-const app = require("../app.js");
+const { app } = require("../app.js");
+const mongoose = require('mongoose');
+let Profile = require('../models/Profile.js');
+
+
 
 chai.should();
 chai.use(chaiHttp);
 
-describe("/POST ping", () => {
+describe('Profiles', function () {
+  beforeEach((done) => {
+    Profile.remove({}, (err) => {
+       done();
+    });
+});
+
+// 
+// /auth/logout test
+//
+
+describe("GET /auth/logout", () => {
   it("it should return 200 and message", (done) => {
     chai
       .request(app)
-      .post(`/ping/`)
-      .send({ message: "hello" })
+      .post(`/auth/logout`)
       .end((err, res) => {
         res.should.have.status(200);
-        res.body.should.have
-          .property("response")
-          .eql("Server is running. Message received: hello");
+        res.text.should
+          .eql("You have successfully logged out");
         done();
       });
   });
 });
+
+// 
+// /auth/register test
+//
+
+describe("POST /auth/register", () => {
+  it("it should return 200 and message", (done) => {
+    let username = "doingTesting";
+    let email = "testing0101@io.io";
+    let password = "testing";
+
+    chai
+      .request(app)
+      .post(`/auth/register`)
+      .send( {username, email, password},)
+      .end((err, res) => {
+        if(err) return done(err);
+
+        res.should.have.status(201);
+        done();
+      });
+  });
+});
+
+
+// 
+// /auth/login test
+//
+
+describe("POST /auth/login", () => {
+  it("it should return 200 and message", (done) => {
+    let email = "testing@io.io";
+    let password = "testing";
+
+    chai
+      .request(app)
+      .post(`/auth/login`)
+      .send({email , password},)
+      .end((err, res) => {
+        if(err) return done(err);
+
+        res.should.have.status(200);
+        done();
+      });
+  });
+});
+
+
+// 
+// /auth/user test
+//
+
+describe("GET /auth/user", () => {
+  it("it should return 200 and message", (done) => {
+    let user = "doingTesting";
```

Comment:
```suggestion
    const user = "doingTesting";
```

These should be constants

Reply:
would be applicable throughout the file